src/components/teams/Teams.js

```diff
@@ -243,11 +256,32 @@ export default class Teams extends Component {
   _getTeamsNavigation() {
     var teams = this.props.teams.filter(team => team.competition === this.props.competition);
     return teams.map(team =>
-      <NavItem eventKey={team.id}>
+       this._getTeamsNavigationTab(team)
+    );
+  }
+
+  _getTeamsNavigationTab(team) {
+    var dangerZone = team.ranking.length - 2;
+    var ranking = this._renderOwnTeamPosition(team);
+    if (ranking <= 3) {
+      return ( <NavItem eventKey={team.id}>
+          <div>{team.competition + ' ' + team.teamCode}&nbsp;&nbsp;
+          <span className="label label-as-badge label-success">{ranking}</span></div>
+      </NavItem>
+      );
```

Comment:
je herhaalt hier ook weer 2x dezelfde code...
Als je de code checkt, dan zie je echter dat de code identiek is wanneer je de label-success en label-danger eruit haalt.

Het is dus mogelijk om:

```
var style = getLabelClassName(team);
return ( <NavItem eventKey={team.id} style={style}>
           <div>{team.competition + ' ' + team.teamCode}&nbsp;&nbsp;
           <span className="label label-as-badge label-success">{ranking}</span></div>
       </NavItem>
       );

function getLabelClassName(team) {
  if (team.isTopper()) return 'label-success';
  if (team.isInDegradationZone()) return 'label-danger';
  return 'label-default';
}
```